prac_06/guitars.py

```diff
@@ -0,0 +1,24 @@
+from prac_06.guitar import Guitar
```

Comment:
You might want to put a module docstring `"""CP1404"""` at the top of this program?